ui/app/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -148,11 +150,25 @@ export default class SignatureRequestOriginal extends Component {
     )
   }
 
-  renderRequestInfo = () => {
+  renderOriginInfo = () => {
+    const { txData, domainMetadata } = this.props
+
+    const originMetadata = txData.msgParams.origin
+      ? domainMetadata?.[txData.msgParams.origin]
+      : null
+
     return (
-      <div className="request-signature__request-info">
-        <div className="request-signature__headline">
-          {this.context.t('yourSigRequested')}
+      <div className="request-signature__origin-row">
+        <div className="request-signature__origin-label">Origin:</div>
```

Comment:
Should "Origin:" be translated?

Reply:
Of course :facepalm: Yes it should, good catch!

Reply:
Resolved in a774097dd0c41cb4ec56ff1705ca966baf0803de